inference-engine/tests/functional/plugin/cpu/single_layer_tests/ctc_Loss.cpp

```diff
@@ -0,0 +1,254 @@
+                } else if (funcInput.get_element_type() == ElementType::i64) {
+                    auto begin = tensor.data<int64_t>();
+                    std::copy(logitLength.begin(), logitLength.end(), begin);
+                }
+            } else if (i == 2) {
+                std::mt19937 genLable(42);
+                std::uniform_int_distribution<unsigned long> distLabel(0, C - 1);
+                std::vector<int32_t> labels(N * T, 0);
+                for (int n = 0; n < N * T; n++) {
+                    int value;
+                    // make sure blank not be inclded in labels
+                    while ((value = distLabel(genLable)) == blank) {}
+                    labels[n] = value;
+                }
+                tensor = ov::runtime::Tensor{funcInput.get_element_type(), {shapeNT}};
+                if (funcInput.get_element_type() == ElementType::i32) {
+                    auto begin = tensor.data<int32_t>();
+                    std::copy(labels.begin(), labels.end(), begin);
+                } else if (funcInput.get_element_type() == ElementType::i64) {
+                    auto begin = tensor.data<int64_t>();
+                    std::copy(labels.begin(), labels.end(), begin);
+                }
+            } else if (i == 3) {
+                std::mt19937 gen(24);
+                std::uniform_int_distribution<unsigned long> dist(1, T);
+
+                std::vector<int32_t> labelLength(N, 0);
+                for (int n = 0; n < N; n++) {
+                    const int len = dist(gen);
+                    // make sure lableLen <= logitLen
+                    labelLength[n] = std::min(len, logitLength[n]);
+                }
+
+                tensor = ov::runtime::Tensor{funcInput.get_element_type(), {shapeN}};
+                if (funcInput.get_element_type() == ElementType::i32) {
+                    auto begin = tensor.data<int32_t>();
+                    std::copy(labelLength.begin(), labelLength.end(), begin);
+                } else if (funcInput.get_element_type() == ElementType::i64) {
+                    auto begin = tensor.data<int64_t>();
+                    std::copy(labelLength.begin(), labelLength.end(), begin);
+                }
+            }
+            inputs.insert({funcInput.get_node_shared_ptr(), tensor});
+        }
+    }
+
+private:
+    int blank;
+};
+
+TEST_P(CTCLossLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED();
+
+    run();
+    CheckPluginRelatedResults(executableNetwork, "CTCLoss");
+}
+
+namespace {
+    const ngraph::element::TypeVector fPrecisions = {
+        ngraph::element::f32
+        // ngraph::element::f16
+    };
+
+    const ngraph::element::TypeVector iPrecisions = {
+        ngraph::element::i32,
+        ngraph::element::i64
+    };
+
+    const std::vector<bool> preprocessCollapseRepeated = {true, false};
+    // TODO: [ticket:71088] there are some inconsistency between cpu plugin and ng ref
```

Comment:
is it correct issue number?
also better to add such test cases in openvino/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

Reply:
number corrected and all skip moved to skip_tests_config.cpp. thanks!